Survey/Sources/Presentation/Modules/Home/HomeView.swift

```diff
@@ -13,12 +13,36 @@ struct HomeView: View {
 
     @ObservedObject var input: HomeViewModel.Input
     @ObservedObject var output: HomeViewModel.Output
+    @State var currentPage = 0
+    @State var tabSelection = 0
+
+    private let minDragTranslationForSwipe: CGFloat = 60.0
+    private let list = Array(APISurvey.dummyList.enumerated())
 
     var body: some View {
-        VStack {
-            Button("Home", action: {})
-                .accessibilityIdentifier(TestConstants.Home.homeButton)
-        }
+        LoadingView(
+            isShowing: .constant(false),
+            text: .constant(""),
+            content: { geo in
+                ZStack(alignment: .top) {
+                    ScrollView(.horizontal) {
```

Comment:
With the setup with tabViewSetup, you don't need ScrollView at all, I suggest to remove the view to reduce redundancy.